primo/utils/setup_logger.py

```diff
@@ -81,9 +78,28 @@ def setup_logger(
         file_handler = logging.FileHandler(log_file)
         handlers.append(file_handler)
 
+    if log_level in [0, 1, 2]:
```

Comment:
This prints a shorter log message for "INFO" and "WARNING" levels, and prints the full/detailed log message only for the `DEBUG` level.